tests/yam/agama/auto.pm

```diff
@@ -8,15 +8,31 @@ use base 'y2_installbase';
 use strict;
 use warnings;
 
-use testapi 'assert_screen';
-
+use testapi;
+use utils 'clear_console';
 
 sub run {
     assert_screen('agama-main-page', 120);
-    assert_screen('agama-installing', 60);
+    my @tags = qw(agama-installing agama-install-finished  agama-install-reboot welcome-to, login);
+    my $timeout = 600;
+    my ($ext) = get_required_var('AGAMA_AUTO') =~ /(\.[^.]+)$/;
+    while(1) {
```

Comment:
As we have discussed many times offline, code with all this kind of hacks will be not accepted as production code: `while, if, last if, sleep`.

Reply:
done.